manifests/config.pp

```diff
@@ -21,12 +14,12 @@
     owner  => $katello_devel::user,
     group  => $katello_devel::group,
     mode   => '0755',
-  } ->
-  file { "${katello_devel::deployment_dir}/foreman/config/settings.plugins.d/katello.yaml":
-    ensure  => file,
-    content => template('katello/katello.yaml.erb'),
-    owner   => $katello_devel::user,
-    group   => $katello_devel::group,
-    mode    => '0644',
   }
+
+  katello_devel::plugin { 'katello/katello': }
+
+  $extra_plugins.each |String $plugin| {
```

Comment:
Yea, I misread the docs and push to test it locally. I also tested your suggestion and that seemed to not do what I expected :/

```
[ERROR 2016-05-06 10:35:27 verbose]  Could not find template 'katello/.yaml.erb' at /usr/share/katello-installer-base/modules/katello_devel/manifests/plugin.pp:10 on node devel.example.com
```

Failing at this line https://github.com/Katello/puppet-katello_devel/pull/71/files/9922a3b9649bf35750a5d07e4f89bff98b01b702#diff-27f9b7e25f91571b5a1d6dcda89e134cR4